package/README.md

```diff
@@ -1,66 +1,49 @@
-- [Real-time event broadcasting](https://laravel.com/docs/broadcasting).
-
-Laravel is accessible, powerful, and provides tools required for large, robust applications.
-
-## Learning Laravel
-
-Laravel has the most extensive and thorough [documentation](https://laravel.com/docs) and video tutorial library of all modern web application frameworks, making it a breeze to get started with the framework.
-
-You may also try the [Laravel Bootcamp](https://bootcamp.laravel.com), where you will be guided through building a modern Laravel application from scratch.
-
-If you don't feel like reading, [Laracasts](https://laracasts.com) can help. Laracasts contains over 2000 video tutorials on a range of topics including Laravel, modern PHP, unit testing, and JavaScript. Boost your skills by digging into our comprehensive video library.
-
-## Laravel Sponsors
-
-We would like to extend our thanks to the following sponsors for funding Laravel development. If you are interested in becoming a sponsor, please visit the Laravel [Patreon page](https://patreon.com/taylorotwell).
-
-### Premium Partners
-
-- **[Vehikl](https://vehikl.com/)**
-- **[Tighten Co.](https://tighten.co)**
-- **[Kirschbaum Development Group](https://kirschbaumdevelopment.com)**
-- **[64 Robots](https://64robots.com)**
-- **[Cubet Techno Labs](https://cubettech.com)**
-- **[Cyber-Duck](https://cyber-duck.co.uk)**
-- **[Many](https://www.many.co.uk)**
-- **[Webdock, Fast VPS Hosting](https://www.webdock.io/en)**
-- **[DevSquad](https://devsquad.com)**
-- **[Curotec](https://www.curotec.com/services/technologies/laravel/)**
-- **[OP.GG](https://op.gg)**
-- **[WebReinvent](https://webreinvent.com/?utm_source=laravel&utm_medium=github&utm_campaign=patreon-sponsors)**
-- **[Lendio](https://lendio.com)**
-
-## Contributing
-
-Thank you for considering contributing to the Laravel framework! The contribution guide can be found in the [Laravel documentation](https://laravel.com/docs/contributions).
-
-## Code of Conduct
-
-In order to ensure that the Laravel community is welcoming to all, please review and abide by the [Code of Conduct](https://laravel.com/docs/contributions#code-of-conduct).
-
-## Security Vulnerabilities
-
-If you discover a security vulnerability within Laravel, please send an e-mail to Taylor Otwell via [taylor@laravel.com](mailto:taylor@laravel.com). All security vulnerabilities will be promptly addressed.
-
-## License
-
-The Laravel framework is open-sourced software licensed under the [MIT license](https://opensource.org/licenses/MIT).
+<div align="center">
+<h1>Cluster Cache</h1>
+Speed up your application using Cluster Cache 
+</div>
+<hr>
+<br />
+
+## Requirements
+
+* Laravel >= 9.0
+* [SHMOP](https://www.php.net/manual/en/ref.shmop.php)
+
+### Optional Requirements
+* [Igbinary](https://www.php.net/manual/en/book.igbinary.php) - it isn't mandatory but influences on the performance
+
+## Installation
+```bash
+composer require sopamo/clustercache
```

Comment:
Ok, so I finally found out, that it only installed one specific branch (feature/package) because that still had a composer.json in the root of the project. Turns out all packages that get installed via composer need to have a composer.json in their root. Once you change it accordingly, I can finish reviewing the installation :)

Reply:
I think I can switch to PHP 8.0. Laravel 9 supports PHP 8.0

Hmm how do you suggest to  solve issue with `composer.json`? Should we have 2 repos? `project-test` exists mainly to run tests

Reply:
Usually the composer.json is in the same folder (or even higher up) than the folder that contains tests (see kl relaunch for example). You think that would be a problem?

The php doc is not a good source for installation instructions. Generally, you can just install shmop via apt. In a php based docker container, I think you can also install it with a special command to install php extensions. I think we should provide different installation instructions for these two environments.